### PR TITLE
Use React context instead of context prop

### PR DESCRIPTION
**Pull request by mridgway**

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: chat/README.md [hunk 1]
```diff
@@ -10,17 +10,6 @@ Make sure you run `npm install` from the root of flux-examples. Then change dire
 
 ```bash
 $ npm run build
```
Comment (jedireza) [inline chat/README.md]: `npm run dev`

File: chat/components/ThreadSection.jsx [hunk 2]
```diff
@@ -45,12 +45,11 @@ var ThreadSection = React.createClass({
     render: function() {
         var threadListItems = this.state.threads.map(function(thread) {
             return (
-                <NavLink href={"/thread/" + thread.id} context={this.props.context} key={thread.id} >
+                <NavLink href={"/thread/" + thread.id} key={thread.id} >
```
Comment (jedireza) [inline chat/components/ThreadSection.jsx]: Let's clean up that trailing space before `>`

**Comment by jedireza** [comment]
:+1: Just a couple comments. I'm really liking the new context stuff.

**Comment by redonkulus** [comment]
:shipit:

**Pull request merged**